### ISSUE TITLE
IMPORT FOREIGN SCHEMA readonly always 'true'

### ISSUE THREAD

**Issue by jacobdr** [laurenz/oracle_fdw#330]
I think that I am not doing anything wrong but let me know if this is the expected behavior...

```sql
IMPORT FOREIGN SCHEMA "SOME_ORACLE_SCHEMA"
    LIMIT TO (table1, table2)
    FROM SERVER oracle_some_server_same INTO my_postgres_schema OPTIONS (readonly 'false');
```

```
\det+ my_postgres_schema.*
```

The above results in each table having `readonly 'true'`

**Comment by laurenz** [comment]
Thanks for the report!
What an embarrassing (if small) bug.

**Comment by jacobdr** [comment]
@laurenz what an amazing software product. Thanks for the hard work.

**Comment by jacobdr** [comment]
Fixed in 302eb0c30c7c59732a5ee46a076a271e28973335
